backend/open_webui/retrieval/vector/dbs/chroma.py

```
@@ -102,8 +107,7 @@ def query(
                     }
                 )
             return None
-        except Exception as e:
-            print(e)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)